packages/node/src/methods/app-instance/install/controller.ts

```diff
@@ -29,12 +29,13 @@ export default async function installAppInstanceController(
     params
   );
 
-  const installApprovalMsg: NodeMessage = {
+  const installApprovalMsg: InstallMessage = {
     from: this.address,
-    event: Node.EventName.INSTALL,
+    event: NODE_EVENTS.INSTALL,
     data: {
-      appInstanceId: appInstance.id,
-      proposal: false
+      params: {
+        appInstanceId: appInstance.id
```

Comment:
I just realized after having been confused by the variable name here the confusion that arises from the subtle difference between `AppInstance` and `AppInstanceInfo`. I think we should rename all of these variables to `appInstanceInfo` until we find a way to make the distinction cleaner through abstraction.

Namely, `AppInstanceInfo.prototype.id` is a _property_ to fetch a declared UUID style identifier but `AppInstance.prototype.identityHash` (previously `.id`) is a getter and a _computed property_ for the hash of the `AppInstance.prototype.identity` (an `AppIdentity`).

Reply:
Are you specifically referring to `appInstanceId`?

Following previous discussions, the term `appInstanceId` is explicitly said to refer to the ID of the AppInstanceInfo (which you're referring to as `AppInstanceInfo.prototype.id`) and `appInstanceIdentityHash` refers to the computed property which you're referring to as `AppInstance.prototype.identityHash`.

This is explained here: https://github.com/counterfactual/monorepo/blob/master/packages/node/src/db-schema.ts#L12-L30

Reply:
Can you further clarify which variables you're referring to for renaming though? Because the params don't return an `AppInstanceInfo`, they return an `AppInstanceInfo`'s ID.

Reply:
Right, I agree with the parameter `appInstanceId`, but I just mean the variable in the code above named `appInstance` I think we should name `appInstanceInfo` because it is an `AppInstanceInfo` object, not an `AppInstance` object